drivers/hmis_external_apis/public/schemas/form_definition.json

```diff
@@ -94,107 +231,596 @@
+                45,
+                46,
+                47,
+                48,
+                49,
+                50,
+                51,
+                52,
+                53,
+                54,
+                55
+              ]
+            }
           }
         }
       ]
     },
     "initial": {
       "type": "object",
       "additionalProperties": false,
-      "required": ["initial_behavior"],
+      "required": [
+        "initial_behavior"
+      ],
       "properties": {
-        "_comment": { "type": "string" },
-        "initial_behavior": { "type": "string" },
-        "value_local_constant": { "type": "string" },
-        "value_code": { "type": "string" },
-        "value_number": { "type": "integer" },
-        "value_boolean": { "type": "boolean" }
+        "_comment": {
+          "type": "string"
+        },
+        "initial_behavior": {
+          "enum": [
+            "IF_EMPTY",
+            "OVERWRITE"
+          ]
+        },
+        "value_local_constant": {
+          "type": "string"
+        },
+        "value_code": {
+          "type": "string"
+        },
+        "value_number": {
+          "type": "integer"
+        },
+        "value_boolean": {
+          "type": "boolean"
+        }
       }
     },
-    "enable_when": {
+    "enableWhen": {
       "type": "object",
       "additionalProperties": false,
       "properties": {
-        "_comment": { "type": "string" },
-        "question": { "type": "string" },
-        "compare_question": { "type": "string" },
-        "local_constant": { "type": "string" },
-        "operator": { "type": "string" },
-        "answer_code": { "type": "string" },
-        "answer_codes": { "type": "array", "items": { "type": "string" } },
-        "answer_number": { "type": "integer" },
-        "answer_boolean": { "type": "boolean" },
-        "answer_group_code": { "type": "string" }
+        "_comment": {
+          "type": "string"
+        },
+        "question": {
+          "$ref": "#/$defs/linkId"
+        },
+        "compare_question": {
+          "$ref": "#/$defs/linkId"
+        },
+        "local_constant": {
+          "type": "string"
+        },
+        "operator": {
+          "enum": [
+            "EQUAL",
+            "EXISTS",
+            "ENABLED",
+            "IN",
+            "INCLUDES",
+            "GREATER_THAN",
+            "GREATER_THAN_EQUAL",
+            "LESS_THAN",
+            "LESS_THAN_EQUAL",
+            "NOT_EQUAL"
+          ]
+        },
+        "answer_code": {
+          "type": "string"
+        },
+        "answer_codes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "answer_number": {
+          "type": "integer"
+        },
+        "answer_boolean": {
+          "type": "boolean"
+        },
+        "answer_group_code": {
+          "type": "string"
+        }
       }
     },
-    "autofill_values": {
+    "autofillValues": {
```

Comment:
This needs one-of logic as annotated here https://github.com/greenriver/hmis-warehouse/blob/087bbb6f3398bc90430bc7ea16562140170e5222/drivers/hmis/app/graphql/types/forms/autofill_value.rb#L14-L20